packages/adapter-prisma/src/index.ts

```diff
@@ -310,45 +309,47 @@ export function PrismaAdapter(
         .create({
           data: authenticator,
         })
-        .then(fromDBAuthenticator)
+        .then((authenticator) => {
+          const { id, ...rest } = authenticator
+          return {
+            ...rest,
+          }
+        })
     },
     async getAuthenticator(credentialID) {
       const authenticator = await p.authenticator.findUnique({
         where: { credentialID },
       })
-      return authenticator ? fromDBAuthenticator(authenticator) : null
+      if (!authenticator) return null
+      const { id, ...rest } = authenticator
+      return {
+        ...rest,
+      }
     },
     async listAuthenticatorsByUserId(userId) {
       const authenticators = await p.authenticator.findMany({
         where: { userId },
       })
 
-      return authenticators.map(fromDBAuthenticator)
+      return authenticators.map((authenticator) => {
+        const { id, ...rest } = authenticator
+        return {
+          ...rest,
+        }
+      })
     },
     async updateAuthenticatorCounter(credentialID, counter) {
       return p.authenticator
         .update({
           where: { credentialID: credentialID },
           data: { counter },
         })
-        .then(fromDBAuthenticator)
+        .then((authenticator) => {
+          const { id, ...rest } = authenticator
```

Comment:
Q: why does the id have to be omitted in the first place? Is this being shipped to the client so it's omitted for security reasons?

Reply:
Good question, wasn't involved in the writing of this part, but I assume it's just because its internal info that isn't required externally. 

That `id` is just the Prisma / whatever DB primary key in the table.

Afaik there's no lookup by that ID that a user could execute or anythign like that.

The security here comes from the webauthn private key that the user has and the public key that auth.js / this table contain. Not anything related to this `id`, right.

Sooo long story short, _I think_ we can probably also drop this part and return the `id`. What do you think?

Reply:
Ehm if it's not used, why is it required in the db schema? Can we not omit it and have a different PK similar to account model?

Reply:
So theoretically yeah. The author probably went with a separate `id` column just because that's the common first thing to do.

I like the idea of iterating here and removing the unnecessary column, I'm just worried about breaking changes for folks potentially already running this. Although that being said, webauthn support isn't officially even announced and labelled as experimental on top of an already "beta" `5.x` release, so I guess it wouldn't be that big of a deal

Reply:
Okay so I updated the `authenticator` schema to have the `credentialId + userId` as the primary, and updated docs/types/tests.

I'll double check it tomorrow, but should be okay. 

Migration-wise it actually shouldn't be tough for folks that may have already used Passkeys cause we weren't using the `id` field at all.